Documentation/block.md

```diff
@@ -99,3 +105,49 @@ kubectl delete -f mysql.yaml
 kubectl delete -n rook-ceph pool replicapool
 kubectl delete storageclass rook-ceph-block
 ```
+
+## Advanced Example: Erasure Coded Block Storage
+
+If you want to use erasure coded pool with RBD, you need to create two pools (as seen below): one erasure coded and one replicated.
+You need to specify the replicated pool as the `pool` parameter. It is used for the metadata of the images.
+The erasure coded pool must be set as the `dataPool` parameter below. It is used for the data of the images.
+
+```yaml
+apiVersion: ceph.rook.io/v1beta1
+kind: Pool
+metadata:
+  name: replicated-metadata-pool
+  namespace: rook-ceph
+spec:
+  replicated:
+    size: 3
+---
+apiVersion: ceph.rook.io/v1beta1
+kind: Pool
+metadata:
+  name: ec-data-pool
+  namespace: rook-ceph
+spec:
+  # Make sure you have enough nodes and OSDs running bluestore to support the replica size or erasure code chunks.
+  # For the below settings, you need at least 3 OSDs on different nodes (becauase the `failureDomain` is `host` by default).
```

Comment:
- the bluestore OSDs requirement is only for EC pools
- I was under the impression that in order to have EC pool you need to have **all** the OSDs running bluestore (I remember someone Slack saying that it still gets an error trying to create an 3-chunked EC pool with 7 bluestore and 2 filestore OSDs)
- s/becauase/because

Reply:
1. This is the manifest/definition for an erasure coded Pool that is why the comments are here. They are only meant for this Pool. The "normal" replicated Pool is the first object in this snippet.
2. Normally, you only need enough bluestore OSDs that are applicable for placement (e.g. dependent on the `failureDomain`).
3. Fixed.